CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 1.0.3
+* Add support for configuration maps for Custom Rule Factories ([#1122](https://github.com/joelittlejohn/jsonschema2pojo/pull/1122))
+
```

Comment:
No need to add this btw. It gets generated automatically from GitHub issues when we release.